server/helios/models.py

```diff
@@ -915,14 +916,27 @@ def get_user(self):
     def register_user_in_election(cls, user, election):
         voter_uuid = str(uuid.uuid4())
         voter = Voter(uuid=voter_uuid, user=user, election=election)
-
+        
         # do we need to generate an alias?
         if election.use_voter_aliases:
             heliosutils.lock_row(Election, election.id)
             alias_num = election.last_alias_num + 1
             voter.alias = "V%s" % alias_num
 
         voter.save()
+        qr_code_content = "content"
```

Comment:
Add TODO to fix `content` with proper data

Reply:
Done